### DISCUSSION TITLE
How can I access images in a local storage bucket?

### DISCUSSION THREAD

**Discussion by clemwo** [supabase/supabase#29035] [General]
I have uploaded images locally in the local storage bucket that is set to public. 
Now when I open the the web interface and click `Get URL` on the specific image I get the following url which looks alright: 
http://localhost/storage/v1/object/public/option-images/a9cea0a4-b49c-4580-800a-a2cae8a6fc23/image.jpg

However when I click on it, I get the error `Unable to connect` in my webbrowser.

Shouldn't the image be available like that?

**Comment by gc-ft** [comment]
Seems to be missing the port which is 54321 by default...

So http://localhost:54321/storage/v1/object/public/option-images/a9cea0a4-b49c-4580-800a-a2cae8a6fc23/image.jpg

**Reply by clemwo** [reply]
Cool, that solves it. I already tried adding the port but apparently messed that up. Thank you so much for the quick reply :)
